tests/test_mssqlcliclient.py

```diff
@@ -240,7 +242,20 @@ def test_mssqlcliclient_multiple_statement(client, query_str, rows_outputted):
         for (i, (rows, _, _, query_executed, is_error)) in \
             enumerate(client.execute_query(query_str)):
 
-            queries = query_str.split(';')
+            queries = ["{};".format(query) for query in query_str.split(';')]
```

Comment:
If a literal string has a semi-colon would this parse? Looks like an existing bug so this doesn't block this change. Can we add a test for this an confirm, and file a bug if needed?

Reply:
This is just our testing functionality—semicolons parse just fine in the product code. We could hard-code the array if we want to add a test with semicolons.

Reply:
oh this is just a test - my bad  :)

Reply:
However, are we testing literal strings :)  :)